### PR TITLE
[Symfony43] Add missing rule WebTestCaseAssertIsSuccessfulRector #249

### PR DESCRIPTION
**Pull request by JohJohan**
See #249

### CODE REVIEW

**Comment by JohJohan** [comment]
I dont know why ECS fails while just adding `WebTestCaseAssertIsSuccessfulRector`


I tried installing vendor packages to check if i could fix it but i got following error:
```bash
You are running Composer with SSL/TLS protection disabled.
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires rector/rector-src dev-main -> satisfiable by rector/rector-src[dev-main].
    - rector/rector-src dev-main requires rector/rector-symfony dev-main -> satisfiable by rector/rector-symfony[dev-main, 0.11.x-dev (alias of dev-main)] from composer repo (https://repo.packagist.org) but rector/rector-symfony is the root package and cannot be mod
ified. See https://getcomposer.org/dep-on-root for details and assistance.
```

**Comment by JohJohan** [comment]
Btw i'm participating hacktober more info here: https://hacktoberfest.com/ and i was wondering if a label could be added to the project or this pull request? `HACKTOBERFEST-ACCEPTED`

File: config/sets/symfony/symfony43.php [hunk 1]
```diff
@@ -12,11 +12,13 @@
 use Rector\Renaming\Rector\Name\RenameClassRector;
 use Rector\Renaming\ValueObject\MethodCallRename;
 use Rector\Symfony\Rector\MethodCall\MakeDispatchFirstArgumentEventRector;
+use Rector\Symfony\Rector\MethodCall\WebTestCaseAssertIsSuccessfulRector;
 use Rector\Symfony\Rector\MethodCall\WebTestCaseAssertResponseCodeRector;
 
 # https://github.com/symfony/symfony/blob/4.4/UPGRADE-4.3.md
 return static function (RectorConfig $rectorConfig): void {
     # https://symfony.com/blog/new-in-symfony-4-3-better-test-assertions
+    $rectorConfig->rule(WebTestCaseAssertIsSuccessfulRector ::class);
```
Comment (samsonasik) [inline config/sets/symfony/symfony43.php:21]: surplus space before `::` need to be removed:

```suggestion
    $rectorConfig->rule(WebTestCaseAssertIsSuccessfulRector::class);
```

Comment (JohJohan) [reply config/sets/symfony/symfony43.php]: Done

**Comment by JohJohan** [comment]
> Please run:
> 
> ```
> vendor/bin/rector && composer fix-cs 
> ```
> 
> to make CI green.

I will have a look as i'm having some problems on windows like:
```
$ vendor/bin/rector


 [ERROR] Could not resolve argument "Rector\Config\RectorConfig $rectorConfig" for
         "x\rector-symfony\vendor\rector\rector-src\src\Kernel/../../config/config.php".


```

**Comment by JohJohan** [comment]
@samsonasik i applied fixes, these changes are needed because we now fetch the latest ecs version? Other i dont understand why these changes are needed if i just add on new rule to `symfony43.php`

**Comment by samsonasik** [comment]
Yes, that because require latest ecs

**Review by samsonasik — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Comment by JohJohan** [comment]
> Btw i'm participating hacktober more info here: https://hacktoberfest.com/ and i was wondering if a label could be added to the project or this pull request? `HACKTOBERFEST-ACCEPTED`

@TomasVotruba can you consider the above comment

**Comment by TomasVotruba** [comment]
@JohJohan Sure :+1: I recall doing my first Hacktober fest, so I'm happy to help :)

**Pull request merged**